src/Middleware/DetectLocaleMiddleware.php

```diff
@@ -42,6 +42,22 @@ class DetectLocaleMiddleware implements HTTPMiddleware
      */
     private static $persist_cookie_expiry = 90;
 
+    /**
+     * Use this path when setting the locale cookie
+     *
+     * @config
+     * @var int
```

Comment:
Could you please use a doc block hint that describes what the property would be? E.g. here it's probably `@var string`

Reply:
Yeah it's a string not an int isn't it?